src/Gui/propertyeditor/PropertyItem.cpp

```diff
@@ -2545,19 +2545,17 @@ QVariant PropertyStringListItem::toString(const QVariant& prop) const
     }
 
     QString text = QString::fromUtf8("[%1]").arg(list.join(QLatin1String(",")));
-    text.replace(QString::fromUtf8("'"),QString::fromUtf8("\\'"));

```

Comment:
I'm just not sure about this... Is it safe?

Reply:
It is safe. `PropertyStringListItem::toString` manages what we "see" in the property editor, not the value of the property itself, so there is no need to replace characters there.